Core/clim-core/panes/stream-panes.lisp

```diff
@@ -410,16 +410,24 @@ current background message was set."))
                           nil)))
       (values pane stream))))
 
-(defun make-clim-interactor-pane (&rest options)
+(defun make-clim-interactor-pane (&rest options &key (scroll-bar t scroll-bar-p) (scroll-bars scroll-bar scroll-bars-p) &allow-other-keys)
+  (unless (or scroll-bar-p scroll-bars-p)
+    (setf (getf options :scroll-bars) :vertical))
   (apply #'make-clim-stream-pane :type 'interactor-pane options))
```

Comment:
This results in a warning because `scroll-bars` is not used.

I also find a non-mutating version easier to understand:
```lisp
(apply #'make-clim-stream-pane :type 'interactor-pane
       (if (or scroll-bar-p scroll-bars-p)
           options
           (list* :scroll-bars :vertical options)))
```
But that may just be my taste.

Reply:
done